test/tools/fuzzer_common.cpp

```diff
@@ -16,7 +16,7 @@
 */
 // SPDX-License-Identifier: GPL-3.0
 
-#include "libsolidity/formal/ModelCheckerSettings.h"
+#include "libsmtutil/SolverInterface.h"
```

Comment:
Why is the include using quotes?

Reply:
That was automatic from clangd, will remvoe